### DISCUSSION TITLE
Migrate 'kothic' to Python 3

### DISCUSSION THREAD

**Discussion by strump** [organicmaps/organicmaps#1827] [Ideas]
Currently `tools/kothic` submodule points to [`vng-fix`](https://github.com/organicmaps/kothic/tree/vng-fix) branch. Maybe it's time to merge the branch into `master` which is already migrated to Python 3?

**Comment by biodranik** [comment]
Unfortunately, it wasn't fully migrated. There were some errors in the code that should be fixed first. It would be great if someone can help and fix it.
